docs/reference/backends.md

```diff
@@ -258,6 +258,7 @@ Current implemented algorithms:
 - `roundRobin`: backend is chosen by the round robin algorithm, starting with a random selected backend to spread across all backends from the beginning
 - `random`: backend is chosen at random
 - `consistentHash`: backend is chosen by a consistent hashing algorithm with the client X-Forwarded-For header with remote IP as the fallback as input to the hash function
+- `powerOfRandomNChoices`: backend is chosen by powerOfRandomNChoices algorithm with selecting N random endpoints and picking the one with least outstanding requests from them. (http://www.eecs.harvard.edu/~michaelm/postscripts/handbook2001.pdf)
```

Comment:
s/endpoints/backends/
s/outstanding/inflight/

Also it may worth mentioning that inflight request are counted per route, i.e. the same backend may have different inflight request count in different routes.